src/order-radix.c

```diff
@@ -2754,6 +2757,47 @@ void cpl_order(SEXP x,
   }
 }
 
+/*
+ * Normalises a complex value so that if one side is missing, both are. This
+ * ensures that all missing complex values are grouped together, no matter
+ * what type of missingness it is. NA and NaN can still be separated by
+ * `nan_distinct`, resulting in 4 different combinations of missingness. These
+ * 4 groups of missingness will still all be grouped together, either before
+ * or after any non-missing values have appeared.
+ * See issue #1403 for more information.
+ */
+static inline
+r_complex_t cpl_normalise(r_complex_t x) {
```

Comment:
Maybe `cpl_normalise_na()`? I tend to avoid "normalise"/"standardise" without mentioning the context or standard against which an input is normalised.